README.md

```diff
@@ -140,6 +140,43 @@ assessment, err := client.RegisterLogin(&incognia.Login{
 })
 ```
 
+### Registering Payment or Login without evaluating its risk assessment
+
+To register a login or a payment without evaluating its risk assessment, you should use the `Eval *bool` atribute as follows:
+
+Login example:
+```go
+shouldEval := false
+
+assessment, err := client.RegisterLogin(&incognia.Login{
+    Eval:           &shouldEval,
+    InstallationID: "installation-id",
+    AccountID:      "account-id",
+    ExternalID:     "external-id",
+})
+```
+
+Payment example:
+```go
+shouldEval := false
+
+assessment, err := client.RegisterPayment(&incognia.Payment{
+    Eval:            &shouldEval,
+    InstallationID: "installation-id",
+    AccountID:      "account-id",
+    ExternalID:     "external-id",
+    Addresses: []*incognia.TransactionAddress{
+        {
+            Type: incognia.Billing,
+            AddressLine:    "20 W 34th St, New York, NY 10001, United States",
+            StructuredAddress: &incognia.StructuredAddress{
+                Locale:       "en-US",
+                CountryName:  "United States of America",
+    ...
+```
+
+Turning off the risk assessment evaluation allows you to register a new transaction (Login or Payment), but the response will be an empty response (`TransactionAssessment` and `SignupAssessment`). For instance, if you're using the risk assessment only for some payment transactions you still should register all the other ones: this will avoid any bias on the risk assessment computation.
```

Comment:
Shouldn't this description be next to the topic name, instead of after the examples?